deployment-templates/py/forseti-instance.py

```diff
@@ -75,20 +93,6 @@ def GenerateConfig(context):
         'properties': {
             'zone': context.properties['zone'],
             'machineType': (
```

Comment:
I got an error running this DM template. Looks like this dict property isn't quite correct?

- code: MANIFEST_EXPANSION_USER_ERROR
  message: |
    Manifest expansion encountered the following errors: Error compiling Python code: invalid syntax (<string>, line 96) Resource: forseti-instance.py Resource: config